test/elixir_google_scraper/scraper/schemas/search_response_test.exs

```diff
@@ -0,0 +1,39 @@
+defmodule ElixirGoogleScraper.Scraper.Schemas.SearchResultTest do
```

Comment:
the file name should be `test/elixir_google_scraper/scraper/schemas/search_result_test.exs` 👍